tycho-testing-harness/src/main/java/org/eclipse/tycho/test/AbstractTychoIntegrationTest.java

```diff
@@ -174,7 +174,7 @@ protected String getMavenHome() {
         return mavenHome;
     }
 
-    protected String getTychoVersion() {
+    protected static String getTychoVersion() {
```

Comment:
It does not make much sense to have such a (protected) method when it only delegates to a public other static method as no one can ever override this....

Reply:
Reverted that as well.